stable/opa/templates/NOTES.txt

```diff
@@ -2,7 +2,7 @@ Please wait while the OPA is deployed on your cluster.
 
 For example policies that you can enforce with OPA see https://www.openpolicyagent.org.
 
-You can query OPA to see the policies it has loaded:
+You can query OPA to see the policies it has loaded (you will need to turn off authz as described below):
```

Comment:
Let's  delete this part from here and move it below. Basically remove lines 5-12.